tweet-api/api/inference.py

```diff
@@ -0,0 +1,44 @@
+import csv
+import urllib.request
+
+import numpy as np
+from scipy.special import softmax
+from transformers import AutoModelForSequenceClassification, AutoTokenizer
+
+task='emotion'
+MODEL = f"cardiffnlp/twitter-roberta-base-{task}"
+tokenizer = AutoTokenizer.from_pretrained(MODEL)
+
+# download label mapping
+mapping_link = f"https://raw.githubusercontent.com/cardiffnlp/tweeteval/main/datasets/{task}/mapping.txt"
+with urllib.request.urlopen(mapping_link) as f:
+    html = f.read().decode('utf-8').split("\n")
+    csvreader = csv.reader(html, delimiter='\t')
+labels = [row[1] for row in csvreader if len(row) > 1]
+
+def _preprocess(text):
+    new_text = []
+    for t in text.split(" "):
```

Comment:
Any particular reason why we use `.split(" ")` instead of `.split()` ?

Reply:
no, good catch!